config/crd/bases/kubekey.kubesphere.io_clusters.yaml

```diff
@@ -255,6 +255,8 @@ spec:
                     type: array
                   privateRegistry:
                     type: string
+                  namespaceOverride:
```

Comment:
This field also can be removed. The reason is the same as above.

Reply:
done